### PR TITLE
time slice 1 min granularity docs

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

https://datadoghq.atlassian.net/browse/DOCS-7519

### CODE REVIEW

**Comment by roxanne-moslehi** [comment]
@estherk15 this one is ready to review and finalize, but we will need to wait for the release happening in the next few days before merging. thanks!

also FYI @peachprosecutor

**Review by estherk15 — APPROVED** [review]
Looks good! Thanks Roxanne!

**Comment by roxanne-moslehi** [comment]
@estherk15 this is ready to merge

**Pull request merged**
